common/static/common/js/common_libraries.js

```diff
@@ -1,22 +0,0 @@
-define([
-    'edx-ui-toolkit/js/utils/string-utils',
-    'edx-ui-toolkit/js/utils/html-utils',
-    'domReady!',
-    'jquery',
-    'jquery-migrate',
-    'backbone',
-    'underscore',
-    'gettext'
-],
-    function(StringUtils, HtmlUtils) {
-        'use strict';
-
-        // Install utility classes in the edX namespace to make them
-        // available to code that doesn't use RequireJS,
-        // e.g. XModules and XBlocks.
-        if (window) {
-            window.edx = window.edx || {};
-            window.edx.StringUtils = StringUtils;
```

Comment:
Good question. We load global-loader.js first which means they get loaded onto ``window.edx``. Studio used to manually install them into this location in common_libraries.js, but that is no longer necessary.